pages/search.js

```diff
@@ -0,0 +1,92 @@
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import { Tabs, Tab, Box, Container } from '@material-ui/core';
+import { useRouter } from 'next/router';
+import Head from 'next/head';
+import getConfig from 'next/config';
+import { t } from 'ttag';
+import querystring from 'querystring';
+import ArticlePageLayout from 'components/ArticlePageLayout';
+import AppLayout from 'components/AppLayout';
+import withData from 'lib/apollo';
+
+const {
+  publicRuntimeConfig: { PUBLIC_URL },
+} = getConfig();
+
+const useStyles = makeStyles(theme => ({
+  jumbotron: {
+    position: 'absolute',
+    background: '#202020',
+    width: '100vw',
+    minWidth: '100vw',
+    left: 0,
+  },
+  search: {
+    color: theme.palette.common.white,
+  },
+  content: {
+    paddingTop: 230,
+  },
+}));
+
+const CustomTab = withStyles(theme => ({
+  root: {
+    color: theme.palette.common.white,
+    '&$selected': {
+      color: theme.palette.primary,
+    },
+  },
+}))(Tab);
+
+function SearchPage() {
+  const router = useRouter();
+  const queryString = querystring.stringify(router.query);
+  const { query } = router;
+
+  const classes = useStyles();
+
+  const navigate = type =>
+    router.push({ pathname: '/search', query: { ...query, type } });
+
+  return (
+    <AppLayout>
+      <Head>
+        <title>{t`Search`}</title>
+        <link
+          rel="alternate"
+          type="application/rss+xml"
+          href={`${PUBLIC_URL}/api/articles/rss2?${queryString}`}
+        />
+        <link
+          rel="alternate"
+          type="application/atom+xml"
+          href={`${PUBLIC_URL}/api/articles/atom1?${queryString}`}
+        />
+      </Head>
+      <div className={classes.jumbotron}>
+        <Container>
+          <Box py="56px">
+            <h2 className={classes.search}>{t`Search`}</h2>
+          </Box>
+        </Container>
+        <Container>
+          <Tabs
+            value={query.type}
+            onChange={(e, page) => navigate(page)}
+            indicatorColor="primary"
+            textColor="primary"
+            aria-label="tabs"
+          >
+            <CustomTab value="messages" label={t`Messages`} />
+            <CustomTab value="replies" label={t`Replies`} />
+          </Tabs>
+        </Container>
+      </div>
+      <div className={classes.content}>
+        <ArticlePageLayout q={{ [query.type]: query.q }} />
```

Comment:
Search result for replies is different from mockup. It should be listing replies that has the query text, rather than articles that has the query text.

Mockup:
![image](https://user-images.githubusercontent.com/108608/80891388-3a872400-8cf6-11ea-9ecc-dfd781fd27fc.png)

Current:
![image](https://user-images.githubusercontent.com/108608/80891392-4377f580-8cf6-11ea-99c0-fae928ed5f47.png)


This may be huge so can include in future PRs.

Reply:
card added here: https://github.com/orgs/cofacts/projects/5#card-34990256